assets/src/design-system/components/menu/useDropDownMenu.js

```diff
@@ -73,9 +74,14 @@ export default function useDropDownMenu({
         focusedIndex < listLength - 1
       ) {
         handleMoveFocus(1);
+      } else if (
+        ['ArrowUp', isRTL ? 'ArrowRight' : 'ArrowLeft'].includes(key) &&
+        focusedIndex === 0
+      ) {
+        handleReturnToParent?.();
```

Comment:
If `handleReturnToParent` is present this allows us an escape hatch to return to the input tied to a menu